src/core/Logo.tsx

```diff
@@ -1,29 +1,65 @@
 import React from "react";
+import LogoAssetMonoStacked from "./images/logo/ably-logo-mono-stacked.svg";
+import LogoAssetMonoWhiteStacked from "./images/logo/ably-logo-mono-white-stacked.svg";
+import LogoAssetMonoWhite from "./images/logo/ably-logo-mono-white.svg";
+import LogoAssetMono from "./images/logo/ably-logo-mono.svg";
+import LogoAssetStacked from "./images/logo/ably-logo-stacked.svg";
+import LogoAssetWhiteStacked from "./images/logo/ably-logo-white-stacked.svg";
+import LogoAssetWhite from "./images/logo/ably-logo-white.svg";
+import LogoAsset from "./images/logo/ably-logo.svg";
 
 type LogoProps = {
-  dataId: string;
-  logoUrl: string;
+  dataId?: string;
+  logoUrl?: string;
+  logoAlt?: string;
   href?: string;
   additionalImgAttrs?: object;
   additionalLinkAttrs?: object;
+  theme?: "light" | "dark";
+  variant?: "default" | "mono";
+  orientation?: "default" | "stacked";
 };
 
 const Logo = ({
   dataId,
-  logoUrl,
   href = "/",
   additionalImgAttrs,
   additionalLinkAttrs,
-}: LogoProps) => (
-  <a
-    href={href}
-    data-id={dataId}
-    className="block"
-    style={{ height: "2.125rem" }}
-    {...additionalLinkAttrs}
-  >
-    <img src={logoUrl} width="108px" alt="Ably logo" {...additionalImgAttrs} />
-  </a>
-);
+  theme = "light",
+  variant = "default",
+  orientation = "default",
+  logoUrl,
+  logoAlt = "Ably logo",
+}: LogoProps) => {
+  const getLogoSrc = React.useCallback(() => {
+    if (logoUrl) return logoUrl;
+
+    if (theme === "dark") {
+      if (variant === "mono") {
+        return orientation === "stacked"
+          ? LogoAssetMonoWhiteStacked
+          : LogoAssetMonoWhite;
+      } else {
+        return orientation === "stacked"
+          ? LogoAssetWhiteStacked
+          : LogoAssetWhite;
+      }
+    } else {
+      if (variant === "mono") {
```

Comment:
if we do this??
```
return variant === "mono"
        ? orientation === "stacked"
          ? LogoAssetMonoStacked
          : LogoAssetMono
        : orientation === "stacked"
        ? LogoAssetStacked
        : LogoAsset;
```

Reply:
It's more condensed, but it doesn't read as clearly to me - I try to avoid nested ternary operations where possible